Trap Quest.inform/Inform/Extensions/Player/Exposure Addiction.i7x

```diff
@@ -0,0 +1,51 @@
+A time based rule (this is the fighting against no-panties fetish rule):
+	if the player is fighting against no-panties:
+		if no panties reminder <= 0:
+			let C be a random worn knickers;
+			if C is knickers, say "[bold type]You [one of][or]still [stopping]feel extremely uncomfortable in your [printed name of C][bold type]!  [one of]You can feel your body desperately wishing to be free of them[if diaper lover > 0] because you only want to wear diapers[end if], and it's affecting your reflexes and making it harder to think properly.[or][stopping][roman type][line break]";
+			now no panties reminder is 30;
+		decrease no panties reminder by 1.
+Report wearing knickers:
+	if the player is fighting against no-panties, say "You immediately feel extremely uncomfortable in these, thanks to your [if diaper lover >= 1]diaper fetish[otherwise]fetish for not wearing panties[end if].".
+This is the knickers can't be summoned with no panties fetish rule:
+	if summoning is 1 and (no-panties permanent fetish is 1 or the class of the player is faerie):
+		unless wearing-target is diaper or (wearing-target is no protection and diaper lover <= 0), rule fails.
+The knickers can't be summoned with no panties fetish rule is listed in the knickers wearability rules.
+
+
+topless permanent fetish is a number that varies.  topless permanent fetish is 0.
+Definition: yourself is fighting against topless:
+	if topless permanent fetish is 1 or there is a worn topless tattoo or the humiliation of the player >= 35000:
+		let C be a random worn bra;
+		if C is bra and C is not fully exposing and C is nipple covering, decide yes;
+		let D be a random worn dress;
+		if D is dress and D is not fully exposing and D is nipple covering, decide yes;
+		let O be a random worn corset;
+		if O is corset and O is nipple exposing, decide yes;
+	decide no.
+braless reminder is a number that varies. braless reminder is 0.
+A time based rule (this is the fighting against topless fetish rule):
+	if the player is fighting against topless:
+		if braless reminder <= 0:
+			let C be a random worn bra;
+			if C is bra, say "[bold type]You [one of][or]still [stopping]feel terrible wearing your [printed name of C][bold type]!  [one of][if there is a worn topless tattoo]Your heart aches and you[else]You[end if] want to free your [BreastDesc]. It lowers your awareness and consumes your thought.[or][stopping][roman type][line break]";
+			now braless reminder is 30;
+		decrease braless reminder by 1.
+Report wearing bra:
+	if the player is fighting against topless, say "You feel very uncomfortable wearing this, [if there is a worn topless tattoo]the tattooed eye in your chest aches[else]because you know they want to see your [BreastDesc][end if].".
+This is the bra can't be summoned with braless fetish rule:
+	if summoning is 1 and (topless permanent fetish is 1 or the class of the player is faerie):
+		unless wearing-target is fully exposing, rule fails.
```

Comment:
I think we want to allow players to wear clothing that obeys the rules or can be top-displaced.  So I think we should change this to something like `unless wearing-target is fully exposing or wearing-target is see-through or wearing-target is not not-top-displacable or wearing-target is nipple exposing, rule fails.`